acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -136,9 +136,14 @@ context('Acceptance tests', () => {
             cy.get('#familienstand_date').clear().type(older_date)
             cy.get('label[for=familienstand_married_lived_separated-no]').click()
             cy.get('div[id=familienstand_confirm_zusammenveranlagung_field]').should('be.visible')
+            cy.get('label[for=familienstand_confirm_zusammenveranlagung]').first().click()
+            cy.get('#familienstand_confirm_zusammenveranlagung').should('be.checked')
             cy.get('#familienstand-0').check()
             cy.get('#familienstand-1').check()
             cy.get('div[id=familienstand_confirm_zusammenveranlagung_field]').should('not.be.visible')
+            cy.get('label[for=familienstand_married_lived_separated-no]').click()
+            cy.get('div[id=familienstand_confirm_zusammenveranlagung_field]').should('be.visible')
+            cy.get('#familienstand_confirm_zusammenveranlagung').should('not.be.checked')
```

Comment:
This tests that the bug you found is no longer occurring, correct?

I am wondering if we either need to introduce similar tests for other fields here as well or if we really need it

Reply:
Correct. Well, there are similar tests, they're just not that explicit (they're assuming that there's nothing in the date field for example). But I have to admit, I'm not sure it's worth the hustle to introduce those tests more explicitly as - the way I understood Nadine - she's planning to split the familienstand page into multiple pages either way.